### PR TITLE
(0.1.8) Update Oceananigans and add tests

### PR DESCRIPTION
**Pull request by simone-silvestri**
PR #52 needs to be corrected. 

The kernel parameters need to be extended as the periodicity of this grid is particular in the north corner, and the halos are extended.

I will add some more tests here.

This code will change again after https://github.com/CliMA/Oceananigans.jl/pull/3894

### CODE REVIEW

**Comment by navidcy** [comment]
resolve conflicts?

**Comment by glwagner** [comment]
> PR https://github.com/CliMA/OrthogonalSphericalShellGrids.jl/pull/52 needs to be corrected.

The tests all passed, so this means we are missing tests.

File: Project.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Adapt = "4"
 KernelAbstractions = "0.9"
 MPI = "0.20"
-Oceananigans = "0.91.3, 0.92 - 0.99"
+Oceananigans = "0.94 - 0.99"
```
Comment (navidcy) [inline Project.toml:18]: there is no v0.94.0 tagged yet, so CI will fail here

**Review by navidcy — APPROVED** [review]

**Comment by simone-silvestri** [comment]
As soon as https://github.com/CliMA/Oceananigans.jl/pull/3894 is in, this PR isn't needed anymore.

**Pull request merged**